src/elements/DipEdge.H

```diff
@@ -135,13 +190,67 @@ namespace impactx::elements
         {
 
             using namespace amrex::literals; // for _rt and _prt
+            using amrex::Math::powi;
```

Comment:
Since `sqrt` is used below w/o a `std::` prefix:
```suggestion
            using namespace std; // for cmath(float)
            using amrex::Math::powi;
```